app/helpers/validation_helper.php

```diff
@@ -4,4 +4,16 @@ function validate_between($check, $min, $max)
     $n = mb_strlen($check);
     return $min <= $n && $n <= $max;
 }
-?>
\ No newline at end of file
+
+function validate_name($string)
+{
+    return preg_match('/^[a-zA-Z -]+$/', $string);
+}
+
```

Comment:
You may use PHP's built in functions for this po instead of creating helpers.
http://php.net/manual/en/function.ctype-alpha.php
http://php.net/manual/en/function.ctype-alnum.php

:bow: